packages/table/src/table.tsx

```diff
@@ -95,7 +95,7 @@ export interface ITableProps extends IProps, IRowHeights, IColumnWidths {
      * index. Note that if you want to update these values when the user
      * drag-resizes a column, you may define a callback for `onColumnWidthChanged`.
      */
-    columnWidths?: number[];
+    columnWidths?: Array<number | null | undefined>;
```

Comment:
Well, hm. It says non-null value in the docs. Should I change that to match as well? More to the point, any reason _not_ to allow null?